samples/bluetooth/fast_pair/locator_tag/sysbuild/ipc_radio/prj_release.conf

```diff
@@ -0,0 +1,23 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+CONFIG_SERIAL=n
+CONFIG_UART_CONSOLE=n
+CONFIG_LOG=n
+
+# Error handling configuration
+CONFIG_RESET_ON_FATAL_ERROR=y
```

Comment:
I think you should also disable:

```
CONFIG_ASSERT=y
CONFIG_DEBUG_INFO=y
CONFIG_EXCEPTION_STACK_TRACE=y
```
These debug options are set by `overlay-bt_hci_ipc.conf`

Reply:
Overwriting the configs may be problematic as `/home/mapi/ncs/nrf/applications/ipc_radio/overlay-bt_hci_ipc.conf` is applied after `/home/mapi/ncs/nrf/samples/bluetooth/fast_pair/locator_tag/sysbuild/ipc_radio/prj.conf`. Maybe we could move these debug options to `prj.conf` in scope of IPC radio app (or create a new debugging overlay)? Asserts should not be necessary to handle Bluetooth HCI IPC.

Reply:
Changed approach after talk with ipc_radio maintainer and resigned from using the sysbuild option that applied the overlay.

Reply:
We can consider it in the follow-up ticket